recipes/arguably/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "arguably" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arguably-{{ version }}.tar.gz
+  sha256: d55963abbab823b8ad1da798216cd6a7106b3238178855d6ad495beed60ab423
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0,<4.0.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.0,<4.0.0
+    - docstring_parser >=0.15.0,<0.16.0
+    - typing-extensions >=4.6.3,<5.0.0
+
+test:
+  imports:
+    - arguably
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/treykeown/arguably
+  doc_url: https://treykeown.github.io/arguably/
+  summary: Turns functions and docstrings into command line interfaces.
+  license: MIT
```

Comment:
I'm not sure about this one, it seems to be using a custom LICENSE:
https://github.com/treykeown/arguably/blob/main/LICENSE.txt

Reply:
looks like the initial part is definitely MIT but they added the weird AI stuff in the middle paragraph.... not sure what to make of it?

Reply:
@conda-forge/core advice needed here

Reply:
While they editing to the license is terrible and, IMO, you should open an issue upstream to point it to them that. Here, all you can do is to add it as a custom license with `LicenseRef-Arguably`.

Reply:
Sigh, maybe we don't wanna deal with this

Reply:
The LicenseRef-Arguably is an "okay" path.